pasta_eln/Extractors/extractor_py.py

```diff
@@ -0,0 +1,22 @@
+"""extract data from
```

Comment:
Can the name of this file be just extractor.py?

Reply:
Sadly, no, since the extractor is a python file that extracts python files. A jpg extractor written in python would in that nomenclature be extractor_jpg.py